include/zenoh_commons.h

```diff
@@ -905,12 +905,14 @@ typedef struct ze_owned_publication_cache_t {
  *     z_keyexpr_t queryable_prefix: The prefix used for queryable
  *     zcu_locality_t queryable_origin: The restriction for the matching queries that will be receive by this
  *                       publication cache
+ *     bool complete: the `complete` option for the queryable
```

Comment:
The option is called `queryable_complete`.

Reply:
done